src/briefcase/commands/update.py

```diff
@@ -91,7 +100,7 @@ def __call__(
         self.finalize(app)
 
         if app:
-            state = self.update_app(
+            return self.update_app(
```

Comment:
As with create - if there's an established pattern in the code, we should follow it consistently.